docs/DeveloperGuide.md

```diff
@@ -159,6 +159,50 @@ Classes used by multiple components are in the `seedu.netconnect.commons` packag
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Person Roles (Employee, Client, and Supplier)
+
+This feature update introduces three new person types into the existing Person model: `Client`, `Supplier`, and `Employee`. These classes extend the base `Person` class and encapsulate various role-specific functionalities and attributes, improving the application's ability to cater to a diverse range of user interactions.
```

Comment:
Should we be using terms like 'new' im not sure about this.

Reply:
Updated!